pkg/workceptor/kubernetes.go

```diff
@@ -990,6 +1075,49 @@ func (kw *kubeUnit) connectToKube() error {
 		return err
 	}
 
+	qps := 100
+	burst := 1000
+
+	// RECEPTOR_KUBE_CLIENTSET_QPS
+	// default: 100
+	envQPS, ok := os.LookupEnv("RECEPTOR_KUBE_CLIENTSET_QPS")
+	if ok {
+		var err error
+		qps, err = strconv.Atoi(envQPS)
+		if err != nil {
+			return fmt.Errorf("invalid value for RECEPTOR_KUBE_CLIENTSET_QPS: %s. Error %s", envQPS, err)
+		}
+		burst = qps * 10
+	}
+
+	// RECEPTOR_KUBE_CLIENTSET_BURST
+	// default: 10 x QPS
+	envBurst, ok := os.LookupEnv("RECEPTOR_KUBE_CLIENTSET_BURST")
+	if ok {
+		var err error
+		burst, err = strconv.Atoi(envBurst)
+		if err != nil {
+			return fmt.Errorf("invalid value for RECEPTOR_KUBE_CLIENTSET_BURST: %s. Error %s", envBurst, err)
+		}
+	}
+
+	kw.config.QPS = float32(qps)
+	kw.config.Burst = burst
+
+	// RECEPTOR_KUBE_CLIENTSET_RATE_LIMITER
+	// default: tokenbucket
```

Comment:
Here https://pkg.go.dev/k8s.io/client-go/rest I can see :


```
	// Rate limiter for limiting connections to the master from this client. If present overwrites QPS/Burst
	RateLimiter flowcontrol.RateLimiter
```

Does that mean that we use default rate limiter and just ignore QPS and BURST values?